agents1/GenericAgent.py

```diff
@@ -382,10 +396,13 @@ def phase_action(self, state):
             res = self.follow_path(state, Phase.DROP_BLOCK)
 
         elif Phase.DROP_BLOCK == self._phase:
-            res = self.drop_block(None)
-            msg = self._mb.create_message(MessageType.DROP_BLOCK,
-                                          block_vis=self._goal_blocks[self._searching_for]["visualization"],
-                                          location=self._goal_blocks[self._searching_for]["drop_off"])
+            if state[self.agent_name]['location'] == self._goal_blocks[self._searching_for]['drop_off']:
```

Comment:
I think we should always tell when we drop a block, it might come in handy for trust, since this is how we could identify the lazy agent